hashtag_scraper/twitterscraper_streamer.py

```diff
@@ -0,0 +1,131 @@
+import datetime as dt
+import logging
+import random
+import time
+
+import pandas as pd
+import twitterscraper
+from langdetect import detect
+from twitterscraper import query_tweets
+
+logger = logging.getLogger('twitterscraper')
+logger.setLevel(logging.WARNING)
+
+HEADERS_LIST = [
+    'Mozilla/5.0 (Windows; U; Windows NT 6.1; x64; fr; rv:1.9.2.13) Gecko/20101203 Firebird/3.6.13',
+    'Mozilla/5.0 (compatible, MSIE 11, Windows NT 6.3; Trident/7.0; rv:11.0) like Gecko',
+    'Mozilla/5.0 (Windows; U; Windows NT 6.1; rv:2.2) Gecko/20110201',
+    'Opera/9.80 (X11; Linux i686; Ubuntu/14.10) Presto/2.12.388 Version/12.16',
+    'Mozilla/5.0 (Windows NT 5.2; RW; rv:7.0a1) Gecko/20091211 SeaMonkey/9.23a1pre'
+]
+
+twitterscraper.query.HEADER = {'User-Agent': random.choice(HEADERS_LIST), 'X-Requested-With': 'XMLHttpRequest'}
+
+
+def detector(s):
+    '''
+    Helper function for detecting language of a given string s using the langdetect module.
+
+            Parameters:
+                    s (str): A string.
+
+            Returns:
+                    detect(s) (str): string of length 2 indicating language of s.
+    '''
+    try:
+        return detect(s)
+    except:
+        None
```

Comment:
show that this return statement is with purpose. Otherwise, change that `None` to `pass`

```suggestion
        return None
```